docs/conf.py

```diff
@@ -398,3 +398,27 @@ def setup(app):
     f.write(zenodo_record)
 
 print(zenodo_record)
+
+
+# -- Creating tutorials.rst ---------------------------------------------------
+
+notebook_list = []
+
+walk = os.walk("io/")
+
+for root, dirs, fnames in walk:
+    for fname in fnames:
+        if fname.endswith(".ipynb") and "checkpoint" not in fname:
+            notebook_list.append("* :doc:`" + root + "/" + fname[:-6] + "`")
+
+notebook_string = ""
+
+for notebook in notebook_list:
+    notebook_string += "\n" + notebook
```

Comment:
I don't see any reason to do it in a separate loop when you can do same thing directly above in L412